tests/test_model_selection.py

```diff
@@ -52,24 +51,17 @@ def test_edge_inconsistency_score(slices, tmp_path):
 
 
 def test_calculate_convex_hull_edge_inconsistency(slices, tmp_path):
-    np.random.seed(0)
-    pairwise_info = partial_pairwise_align(slices[0], slices[1], s=0.7)
+    pairwise_info = partial_pairwise_align(slices[0], slices[1], s=0.7, maxIter=10)
```

Comment:
In a lot of tests, oracle values have been recomputed by using `maxIter` (or `numItermax`) as 10.